<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MySimIIGui::SolverSettings</class>
 <widget class="QWidget" name="MySimIIGui::SolverSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>424</width>
    <height>695</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Solver settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Simulation parameters</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="8" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Output interval steps</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Type</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLineEdit" name="lineEdit_BDF_Order"/>
      </item>
      <item row="9" column="1">
       <widget class="QLineEdit" name="lineEdit_ScalFactor"/>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="lineEdit_MaxIterSteps"/>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="ComboBox_Type"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Min. iteration steps</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Step</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>BDF order </string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Max. iteration steps</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Scaling factor</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_Starting"/>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEdit_Ending"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Starting</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Ending</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>TimeSteppingMethod</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="lineEdit_Step"/>
      </item>
      <item row="8" column="1">
       <widget class="QLineEdit" name="lineEdit_OutputSteps"/>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="lineEdit_MinIterSteps"/>
      </item>
      <item row="6" column="1">
       <widget class="QComboBox" name="ComboBox_TimeSteppingMethod"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
